config/sync/metatag.metatag_defaults.user.yml

```diff
@@ -0,0 +1,12 @@
+uuid: 3aed6970-6e52-4b55-903e-83a673094960
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: MvQPTbQx0Vxwy0ordSHyixdZmLCMpvMdLD69dlwkrKc
+id: user
+label: User
+tags:
+  canonical_url: '[user:url]'
+  description: '[site:name]'
+  title: '[user:display-name] | [site:name]'
```

Comment:
These are indeed the default values. I even put them there because the `cex` command put them there. Otherwise I would not and let metatag simply install them.

Reply:
OK I suggest to remove them now from the list of installed config in the `joinup_seo` module, since they are not defined by us.